compiler-rt/test/fuzzer/out-of-process-fuzz.test

```diff
@@ -14,8 +14,9 @@ RUN: echo %t
 # Out-of-process fuzzing with this rig is slow,
 # we can not wait for the fuzzer to find the faulty input.
 # Just run for a bit and observe the corpus expansion.
-RUN: LIBFUZZER_OOP_TARGET="./oop-target > /dev/null 2>&1 " ./oop-fuzzer -max_len=3 OOP_CORPUS -runs=1000 -jobs=4
+RUN: env LIBFUZZER_OOP_TARGET="./oop-target > /dev/null 2>&1 " ./oop-fuzzer -max_len=3 OOP_CORPUS/seed -runs=1000 -jobs=4 > %t/temp_output.txt 2>&1
 CHECK: Running: OOP_CORPUS/
 CHECK: Running: OOP_CORPUS/
 CHECK: Running: OOP_CORPUS/
-RUN: ./oop-target OOP_CORPUS/* 2>&1 | FileCheck %s
+RUN: ./oop-target OOP_CORPUS/* >> %t/temp_output.txt 2>&1
+RUN: FileCheck %s < %t/temp_output.txt 2>&1
```

Comment:
Why are these changes required? This patch is about updating the use of an environment variable to use `env` The use of the temp file is perhaps fine, but it certainly shouldn't be appended to when only this one command used to run w/ `FileCheck`.